src/common-messagelog/src/main/java/ee/ria/xroad/common/messagelog/MessageLogProperties.java

```diff
@@ -316,14 +320,27 @@ public static int getCleanTransactionBatchSize() {
         return Integer.getInteger(CLEAN_TRANSACTION_BATCH, DEFAULT_CLEAN_TRANSACTION_BATCH_SIZE);
     }
 
-    public static boolean isEncryptionEnabled() {
-        return Boolean.getBoolean(ENCRYPTION_ENABLED);
+    public static boolean getArchiveEncryptionEnabled() {
+        return Boolean.getBoolean(ARCHIVE_ENCRYPTION_ENABLED);
     }
```

Comment:
Boolean getters in the same file seem to use convention isXXXEnabled but here we have getXXXEnabled